docs/tutorials/openfoam/spack-openfoam.md

```diff
@@ -106,24 +103,19 @@ contains the terraform needed to deploy your cluster.
 
 ## Deploy the Cluster
 
-Use the following commands to run terraform and deploy your cluster.
-
+Use below command to deploy your cluster.
 ```bash
-terraform -chdir=spack-openfoam/primary init
-terraform -chdir=spack-openfoam/primary apply
+./ghpc deploy spack-openfoam
```

Comment:
nit. Remove `./` prefix, just `ghpc`

Reply:
After offline discussion with Ivan, updating it to local copy.